### DISCUSSION TITLE
Excluding variable type spec in dfSummary()

### DISCUSSION THREAD

**Discussion by forstem6** [dcomtois/summarytools#194] [Ideas]
Hi,

first, thank you very much for this package. For me it is perfect for presenting quick overviews or analyses. 

One thing I nearly always tweak is getting rid of the variable type specifications. I found your solution on [stackoverflow](https://stackoverflow.com/questions/67721916/getting-rid-of-the-variable-types-in-dfsummary-output-from-summarytools) and it works perfectly.
However, it might be useful to add the functionality as an argument to `dfSummary()`. Unfortunately, I am not skilled enough to add it as a pull request.

Cheers.

**Comment by dcomtois** [comment]
I'm so sorry I never noticed that there were messages in the GH discussion section! I take note of your suggestion. There are big changes coming eventually, it could be the perfect occasion to split that in!
